src/main/java/com/google/sps/data/Keywords.java

```diff
@@ -136,5 +145,22 @@ private static AnalyzeEntitiesResponse analyzeEntity(String message) throws IOEx
       ex.printStackTrace();
       return null;
     }
-  }  
+  }
+  
+  /**
+   * @return a map of keys to their list of keywords of all sets of keywords that the
+   * user defined by id has entered.
+   */
+  private static Map<String, Collection<String>> getKeyKeywordMap(String id) {
```

Comment:
[Nit] I might call this KeyToKeywordMap as I think it's a bit easier to parse without having 'Key' twice in a row.